website/docs/for-react-native/how-to/using-the-portals-plugin.md

```diff
@@ -0,0 +1,143 @@
+---
+title: How To Use The PortalsPlugin
+sidebar_label: Use the PortalsPlugin
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The `PortalsPlugin` provides useful features to aid in communication between your Web and Native applications. It is included in the Ionic Portals library by default and takes advantage of the Capacitor Plugin system.
```

Comment:
We refer to `PortalsPlugin` a lot of places, but the only place that class name is used anymore in a publicly accessible place is in Android. Perhaps we should rename the various "How To Use The PortalsPlugin" docs to something else. The common theme among all the platforms is how to do Pub/Sub and set initial context, so maybe the title could be something along the lines of "How to Communicate with a Portal Web Application"

Reply:
💯 agreed